core/chains/evm/types/models.go

```diff
@@ -76,6 +75,11 @@ func (h *Head) GetParent() commontypes.Head[common.Hash] {
 	return h.Parent
 }
 
+// Return 0 since block difficulty is not relevant for EVM Chains.
+func (h *Head) BlockDifficulty() *utils.Big {
```

Comment:
I am curious - if this is a no-op (non operational function) which function needs this? I dont see this being used in other places? 

I think this might only be used on POW chains? might not be that relevant for solana or cosmos

Reply:
Difficulty is not only for PoW chains!
The field is used for priority in other consensus algorithms and should not be suppressed.

Reply:
ahh i see! thanks for the clarification jordan!